packages/legend-server-sdlc/src/models/configuration/ProjectDependency.ts

```diff
@@ -42,13 +58,13 @@ export class ProjectDependency implements Hashable {
     });
 
     this.projectId = projectId;
-    this.versionId = versionId ?? new VersionId();
+    this.versionId = versionId ?? '0.0.1';
```

Comment:
--> `0.0.0`

Reply:
I used 0.0.1 because i noticed that the default value for the version in the versionId class was 0.0.1